### PR TITLE
added release notes up to 10.18.250.0-UI

### PR DESCRIPTION
**Pull request by BeateRixen**

### CODE REVIEW

File: content/release-2024-0/applications-2024-0.md [hunk 1]
```diff
@@ -33,6 +33,33 @@ Build version</th>
...
+<td>Web SDK</td>
+<td>A new <code>activeClassName</code> input has been added to the data-grid and device-grid components. It can be used to define a class name to be appended to the last clicked row in the grid. Its default value is "active". This option can be deactivated by setting an empty string to the input.</td>
+<td>DM-2324</td>
+<td>10.18.245.0</td>
+<td>UI</td>
+</tr>
+
+<tr>
+<td>Web SDK</td>
+<td>In release 10.16.0.0, core re-usable data-grid-related components and services have been moved to the <code> @c8y/ngx-components</code>. The initial implementations were deprecated and have now been removed.</td>
```
Comment (rssag) [inline content/release-2024-0/applications-2024-0.md:56]:
```suggestion
<td>In release 10.16.0.0, core re-usable data-grid-related components and services have been moved to the <code>@c8y/ngx-components</code>. The initial implementations were deprecated and have now been removed.</td>

```

File: content/release-2024-0/devices-2024-0.md [hunk 2]
```diff
@@ -125,6 +133,23 @@ Build version</th>
...
+<td>The device list now shows complex columns like <code>c8y_SoftwareList</code> correctly after converting them to strings.</td>
+<td>DM-2410</td>
+<td>10.18.226.0</td>
+<td>UI</td>
+</tr>
+
+
+<tr>
+<td>Device management</td>
+<td>Issues with the group filter in device grids resulting in empty result lists have been fixed. The first matching asset is now correctly shown as filter value.</td>
```
Comment (rssag) [inline content/release-2024-0/devices-2024-0.md:147]:
```suggestion
<td>Issues with the group filter in device grids resulting in empty result lists have been fixed. The first matching asset is now correctly shown as the filter value.</td>
```

File: content/release-2024-0/devices-2024-0.md [hunk 3]
```diff
@@ -189,6 +214,14 @@ Build version</th>
 <td>Core</td>
 </tr>
 
+<tr>
+<td>LWM2M</td>
+<td>The LWM2M agent now correctly persists all registration update parameters. Previously, the LWM2M agent did not store changes of registration parameters, for example updated registration lifetimes. This is now fixed.</td>
```
Comment (rssag) [inline content/release-2024-0/devices-2024-0.md:219]:
```suggestion
<td>The LWM2M agent now correctly persists all registration update parameters. Previously, the LWM2M agent did not store changes of registration parameters, for example, updated registration lifetimes. This is now fixed.</td>
```

File: content/release-2024-0/devices-2024-0.md [hunk 4]
```diff
@@ -245,6 +278,15 @@ Build version</th>
 <td>Core</td>
 </tr>
 
+<tr>
+<td>OPC UA</td>
+<td>In OPC UA device gateway nodes, expected but missing information prevented the completion of the address space scan operation. This is now fixed by skipping these nodes with a respective error message in the opcua-device-gateway log files.
```
Comment (rssag) [inline content/release-2024-0/devices-2024-0.md:283]:
```suggestion
<td>In OPC UA device gateway nodes, expected but missing information prevented the completion of the address space scan operation. This is now fixed by skipping these nodes and adding an error message in the opcua-device-gateway log files.
```

File: content/release-2024-0/announcements-2024-0.md [hunk 5]
```diff
@@ -113,25 +101,41 @@ GET /inventory/managedObjects?withLatestValues=true&query=$filter=c8y_LatestMeas
...
-then also that latest values will be affected. 
+then also that latest values will be affected.
+
+
+
+#### Planned
+
+#### Implemented
+
+##### Breaking change in the Alarms, Events, Measurements APIs - required parameters will be introduced
```
[the remark this answers is not in the document]
Comment (BeateRixen) [reply content/release-2024-0/announcements-2024-0.md:130]: This heading is already in place in earlier announcements as "planned" which is why I would want to leave it as is.

**Review by rssag — APPROVED** [review]

**Pull request merged**